prowler/providers/aws/services/inspector2/inspector2_service.py

```diff
@@ -34,54 +33,30 @@ def __batch_get_account_status__(self, regional_client):
                 f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
-    def __list_findings__(self):
-        logger.info("Inspector2 - listing findings...")
+    def __list_active_findings__(self, inspector):
+        logger.info("Inspector2 - Listing active findings...")
         try:
-            for inspector in self.inspectors:
-                try:
-                    regional_client = self.regional_clients[inspector.region]
-                    list_findings_paginator = regional_client.get_paginator(
-                        "list_findings"
-                    )
-                    for page in list_findings_paginator.paginate():
-                        for finding in page["findings"]:
-                            if not self.audit_resources or (
-                                is_resource_filtered(
-                                    finding["findingArn"], self.audit_resources
-                                )
-                            ):
-                                inspector.findings.append(
-                                    InspectorFinding(
-                                        arn=finding["findingArn"],
-                                        region=regional_client.region,
-                                        severity=finding.get("severity"),
-                                        status=finding.get("status"),
-                                        title=finding.get("title"),
-                                    )
-                                )
+            regional_client = self.regional_clients[inspector.region]
+            active_findings = regional_client.list_findings(
+                filterCriteria={
```

Comment:
Is this `filterCriteria` applied at server side or client side?

Reply:
Server side, that is why is way faster now.